.eslintrc.js

```diff
@@ -77,6 +78,18 @@ module.exports = {
     'unicorn/no-array-push-push': 'error',
     'unicorn/prefer-keyboard-event-key': 'error',
     'unicorn/prefer-regexp-test': 'error',
-    'unicorn/prefer-string-replace-all': 'error'
+    'unicorn/prefer-string-replace-all': 'error',
+    // '@intlify/vue-i18n/no-unused-keys': [
```

Comment:
@absidue this can be useful when removing unused keys in the future

Reply:
Vue eslint plugins don't like that we have our templates and javascript in separate files, which is why they complain about false positives (tried turning on some of vue-eslint-plugin's, "no unused" rules in the past and they ended up with mostly false positives because of that, so I turned them off before I ever commited it).

Reply:
There doesn't seem to be too many false positives except when we access a translated value in a weird way (which we probably shouldn't be doing)

ex:
```js
    colorNames: function () {
      return this.colorValues.map(colorVal => {
        // add spaces before capital letters
        const colorName = colorVal.replaceAll(/([A-Z])/g, ' $1').trim()
        return this.$t(`Settings.Theme Settings.Main Color Theme.${colorName}`)
      })
    },
```

and
```js
      const contentType = this.$t('Age Restricted.Type.' + this.contentTypeString)
```

or if the key is in a different file like what we do for the router:
```js
    {
      path: '/settings/profile',
      meta: {
        title: 'Profile.Profile Settings'
      },
      component: ProfileSettings
    }
```